src/core/public/index.ts

```diff
@@ -341,3 +347,5 @@ export {
 };
 
 export { __osdBootstrap__ } from './osd_bootstrap';
+
+export { WorkspacesStart, WorkspacesSetup, WorkspacesService } from './workspace';
```

Comment:
Do we need to export `WorkspacesService`? Doesn't it just get accessed via the core lifecycle methods?

Reply:
You're right, this is not necessary.